ldclient/integrations/__init__.py

```diff
@@ -144,7 +144,8 @@ class Redis:
     def new_feature_store(url: str='redis://localhost:6379/0',
                           prefix: str='launchdarkly',
                           max_connections: int=16,
-                          caching: CacheConfig=CacheConfig.default()) -> CachingStoreWrapper:
+                          caching: CacheConfig=CacheConfig.default(),
+                          **conn_options: Any) -> CachingStoreWrapper:
```

Comment:
Having a single catch-all for a single purpose limits our ability to add more optional parameters in the future, unless they are co-mingled with the conn_options.

It would be preferable to us if you could add a named parameter `redis_opts` which served to hold those pass through configuration options. For now, we would pull those values from max_connections or `redis_opts` if it is set there. We can clean this up in our next major release, but for now, this keeps our ability to make future changes more flexible.